boot/bootutil/src/bootutil_priv.h

```diff
@@ -17,6 +17,10 @@
  * under the License.
  */
 
+/*
+ * Modifications are Copyright  � 2019 Arm Limited.
```

Comment:
Similar comment to the other comments. There is a bit of a conflict here between Apache conventions, and what I believe many companies are going to require. I would be happier if instead of multiple ad-hoc blocks like this, significant contributors would just place an additional copyright line at the top of the file. That is a little harder to figure out how to word when there isn't an existing copyright.

Regardless of Apache origins of this code, we're going to need to come up with a standard way of including copyright markings, or we will be excluding significant contributors (including myself).

Reply:
Thank you, could you share your thoughts with the rest of the MCUBoot community please? It would be useful to start a thread for example on Slack in this topic.

Would your initial suggestion be the one line copyright at the top of the file just like in mcuboot_config.h (mcuboot/boot/zephyr/include/mcuboot_config/mcuboot_config.h) ?

Reply:
I think a one line at the top is fine. I'm not sure what to do with files that don't start out with a copyright. Maybe we need some kind of "For additional copyrights, see NOTICE." Apache doesn't generally put copyrights on individual files, where as many companies will require this.

Reply:
For example in this file's license header there is this sentence:  "See the NOTICE file distributed with this work for additional information regarding copyright ownership. ". However, currently it is required of me to add an explicit copyright to these files. I don't know how could we solve this...